libvirt/tests/src/incremental_backup/incremental_backup_push_multidisks.py

```
@@ -0,0 +1,177 @@
+            disk_obj.add_vm_disk(disk_type, disk_dict, new_image_path)
+        disk_image_list.append(new_image_path)
+        return disk_xml, disk_image_list
+
+    def prepare_guest():
+        """
+        Prepare the guest for backup.
+
+        :return: return the disk images list.
+        """
+        disk_image_list = []
+        disk_dict = eval(params.get("disk_dict", "{}") % "vdb")
```

Ruff (0.14.1)

Use of possibly insecure function; consider using `ast.literal_eval`

(S307)